requires/install.txt

```diff
@@ -1 +1 @@
-tornado>=3.1,<5
+tornado>=6.0.4,<7
```

Comment:
I don't see anything in here that wouldn't support previous versions of tornado. Since this is a library, I would rather us try to support at least tornado 4.2 -> 6 if we can.

Reply:
The `Allow` Header will not be set if the status is `204`.

Reply:
For clarification, @amberheilman Tornado made a change post `4.2` that removed `Allow` header if it was set and the status code is `204` and only fixed it as of `6.0.4`.